testsuite/openshift/objects/api_key.py

```diff
@@ -8,21 +8,22 @@ class APIKey(OpenShiftObject):
     """Represents API Key Secret CR for Authorino"""
 
     @classmethod
-    def create_instance(cls, openshift: OpenShiftClient, name, label, api_key_string):
+    def create_instance(cls, openshift: OpenShiftClient, name, label, api_key, project=None):
         """Creates base instance"""
+        project = project or openshift.project
         model = {
             "apiVersion": "v1",
             "kind": "Secret",
             "metadata": {
                 "name": name,
-                "namespace": openshift.project,
+                "namespace": project,
```

Comment:
I do not fully understand this change, if you wanted to create it in different project, why not simply use different OpenShiftClient?